### PR TITLE
Add infoblock to playing with neon page

### PR DESCRIPTION
**Pull request by bgrenon**
https://neon-next-git-bgrenon-playingwithneon-library-card-neondatabase.vercel.app/docs/get-started-with-neon/signing-up

### CODE REVIEW

File: content/docs/get-started-with-neon/signing-up.md [hunk 1]
```diff
@@ -10,10 +10,20 @@ redirectFrom:
 updatedOn: '2024-08-06T15:23:10.945Z'
 ---
 
-This tutorial guides you through your first steps using Neon as your Postgres database. You'll get familiar with the following concepts:
-
-- How the Neon [object hierarchy](/docs/manage/overview) works, starting with the **Project**.
-- How [branching](https://neon.tech/docs/introduction/branching) keeps your data isolated from production, and how easy it is to reset development branches when ready to start on new work.
+<InfoBlock>
+<DocsList title="What you will learn:">
+<p>The Neon object hierarchy, starting with the **Project**.</p>
```
Comment (danieltprice) [inline content/docs/get-started-with-neon/signing-up.md:15]:
```suggestion
<p>The Neon object hierarchy, starting with the **Project**</p>
```

File: content/docs/get-started-with-neon/signing-up.md [hunk 2]
```diff
@@ -10,10 +10,20 @@ redirectFrom:
...
+<p>How to create dev branches that periodically reset from main.</p>
```
Comment (danieltprice) [inline content/docs/get-started-with-neon/signing-up.md:16]:
```suggestion
<p>How to create dev branches and reset them from main</p>
```

**Review by danieltprice — APPROVED** [review]

File: content/docs/get-started-with-neon/signing-up.md (hunk 1 above)
Comment (danieltprice) [inline content/docs/get-started-with-neon/signing-up.md:15]: How to view and modify data in the console

Comment (bgrenon) [reply content/docs/get-started-with-neon/signing-up.md]: added

**Pull request merged**